### ISSUE TITLE
Github repository out of date

### ISSUE THREAD

**Issue by careys7** [afterpay/afterpay-magento-2#20]
Hi Afterpay! 

Wondering if you could start publishing your releases to Github? It would help us to submit pull requests

**Comment by PivitParkour94** [comment]
Agreed! @careys7 where are you finding new code? We're all hungry ;)

**Comment by careys7** [comment]
This was supplied via email

**Comment by PivitParkour94** [comment]
Ah I see. So they have new versions, we just have to individually get in touch with the Afterpay team. Cool thanks man!
